modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -525,6 +525,19 @@ Note: you must use json to write multi array values.
 | `--output=[OUTPUT]` | The output format to use (`plain`, `json` or `json_pretty`, default is `plain`).
 |===
 
+Examples
+
+Adding redis to the configuration:
+
+[source,console,subs="attributes+"]
+....
+{occ-command-example-prefix} config:system:set \
+   redis \
+   --value '{"host": "192.168.122.9", "port": "6379"}' \
+   --type json
+System config value redis set to json {"host": "192.168.122.9", "port": "6379"}
```

Comment:
This might be misunderstood as being part of the command, if only briefly. I'd suggest adding a line between it and the command.

Reply:
sure

Reply:
Thanks.